### ISSUE TITLE
MissingFile and UnrecognizedStatement errors when trying to run phar

### ISSUE THREAD

**Issue by Rarst** [vimeo/psalm#1094]
I am trying to run psalm via phar and getting following errors (after successfully running `--init` to create the config):

```
>psalm
Scanning files...
Analyzing files...

ERROR: MissingFile - C:\server\phar\psalm.phar:12:1 - Cannot find file C:\server\www\local\phar:\box-auto-generated-alias-8ebef225ae92.phar\.box\bin\check-requirements.php to include
require 'phar://box-auto-generated-alias-8ebef225ae92.phar/.box/bin/check-requirements.php';


ERROR: MissingFile - C:\server\phar\psalm.phar:14:1 - Cannot find file C:\server\www\local\phar:\box-auto-generated-alias-8ebef225ae92.phar\psalm to include
require 'phar://box-auto-generated-alias-8ebef225ae92.phar/psalm';


ERROR: UnrecognizedStatement - C:\server\phar\psalm.phar:16:1 - Psalm does not understand _HumbugBoxf06a61ef170a\PhpParser\Node\Stmt\HaltCompiler
__HALT_COMPILER(); ?>


------------------------------
3 errors found
------------------------------

Checks took 0.25 seconds and used 37.966MB of memory
```

Win7, PHP 7.2.11

```
>psalm --version
Psalm 3.0.1.0@b982090e10b7c860127ee580bd08775049438187
```

**Comment by TysonAndre** [comment]
psalm.phar should be outside of the directory being analyzed, or in a file/directory that's completely excluded from parsing/analysis

**Comment by Rarst** [comment]
> psalm.phar should be outside of the directory being analyzed

It is. `C:\server\phar\psalm.phar` is phar location and `C:\server\www\local\` is current directory where it’s being called.

**Comment by muglug** [comment]
Thanks for this - seems like `Phar::running(false)` uses Unix slashes.

**Comment by muglug** [comment]
I've released Psalm 3.0.2 with the fix
